src/Components/Questions/Questions.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react'
+import { questionsMockData } from '../../data/questionsMockData'
+import '../Questions/Questions.css'
+import { fetchImages } from '../../apiCalls'
+import QuestionCard from '../QuestionCard/QuestionCard'
+import { Link } from 'react-router-dom'
+
+const Questions = () => {
+  const [questions, setQuestions] = useState([])
+  const [images, setImages] = useState([])
+  const [selectedAnswers, updateSelectedAnswers] = useState({
+    FE: [],
+    BE: [],
+  })
+
+  useEffect(() => {
+    fetchImages()
+    .then(data => setImages(data))
+
+    setQuestions(questionsMockData)
+  }, [])
+
+  const handleSelectAnswer = (program, id) => {
+    const oppositeProgram = program === 'FE' ? 'BE' : 'FE'
+    const isQuestionSelected = selectedAnswers[oppositeProgram].includes(id)
+    // debugger
+    const newAnswers = isQuestionSelected
+      ? {
+        ...selectedAnswers,
+        [program]: [...selectedAnswers[program], id]
+      } : {
+        [program]: [...selectedAnswers[program], id],
+        [oppositeProgram]: selectedAnswers[oppositeProgram].filter(savedId => savedId !== id)
+      }
+
+    updateSelectedAnswers(newAnswers)
+  }
+
+  const handleSubmit = () => {
+    // eslint-disable-next-line
+    const winningHouse = selectedAnswers.FE.length > selectedAnswers.BE.length
+      ? 'FE' : 'BE'
+  }
+
+  const singleQuestion = questions.map((question, i) => {
+    return (
+      <QuestionCard
+        key={ i }
+        questionIndex={`${i + 1} / ${questions.length}`}
+        questionProp={question}
+        characterProp={images.length && images[i]}
+        onSelectAnswer={ handleSelectAnswer }
+      />
+    )
+  })
+
+  return (
+    <div className="questions-container">
+      <div>
+      { singleQuestion }
+      </div>
+      <div>
+        { selectedAnswers.FE.length + selectedAnswers.BE.length >= questions.length &&
```

Comment:
I've never made a quiz/test. This makes sense. Great thinking on how to do this.